message_buffer/programs/message_buffer/src/lib.rs

```diff
@@ -84,6 +82,47 @@ pub mod message_buffer {
     ) -> Result<()> {
         instructions::put_all(ctx, base_account_key, messages)
     }
+
+
+    /// Initializes the buffer account with the target_size
+    ///
+    /// TODO: should target_size be a parameter or should
+    ///     it be a hardcoded initial size
+    pub fn create_buffer<'info>(
+        ctx: Context<'_, '_, '_, 'info, CreateBuffer<'info>>,
+        allowed_program_auth: Pubkey,
+        base_account_key: Pubkey,
+        target_size: u32,
+    ) -> Result<()> {
+        instructions::create_buffer(ctx, allowed_program_auth, base_account_key, target_size)
+    }
+
+    pub fn resize_buffer<'info>(
+        ctx: Context<'_, '_, '_, 'info, ResizeBuffer<'info>>,
+        allowed_program_auth: Pubkey,
+        base_account_key: Pubkey,
+        buffer_bump: u8,
+        target_size: u32,
+    ) -> Result<()> {
+        instructions::resize_buffer(
+            ctx,
+            allowed_program_auth,
+            base_account_key,
+            buffer_bump,
+            target_size,
+        )
+    }
+
+    // TODO: should delete_buffer have same CPI restrictions
```

Comment:
@jayantk do we want to be able to call `delete_buffer` outside of a pyth `delete_XXX` ix?

Reply:
yeah let the admin delete whatever.

can you add doc comments to all of these instructions?